model/client4.go

```diff
@@ -518,5 +522,18 @@ func (c *Client4) GetPostsForChannel(channelId string, page, perPage int, etag s
 	}
 }
 
+func (c *Client4) GetPing() (bool, *Response) {
```

Comment:
Hmmm, now that I see this I think it might be better if we create a new model for Ping so that it's easy to extend later into a health check and we don't have to worry about changing the driver function.

So instead of returning a `map[string]interface{}` we return a `model.Ping` from the endpoint and then return that here as well instead of returning a `bool`

What do you think @senk ?